contracts/adapters/AuctionRebalanceExtension.sol

```diff
@@ -73,7 +73,7 @@ contract AuctionRebalanceExtension is BaseExtension {
      * @param _newComponentsAuctionParams   AuctionExecutionParams for new components, indexed corresponding to _newComponents.
      * @param _oldComponentsAuctionParams   AuctionExecutionParams for existing components, indexed corresponding to
      *                                      the current component positions. Set to 0 for components being removed.
-     * @param _shouldLockSetToken           Indicates if the rebalance should lock the SetToken.
+     * @param _shouldLockSetToken           Indicates if the rebalance should lock the SetToken. Has to be false in this version
```

Comment:
```suggestion
     * @param _shouldLockSetToken           Indicates if the rebalance should lock the SetToken.
```

Omit locking restriction from `AuctionRebalanceExtension` natspec

Reply:
@pblivin0x Why should we remove this, it is true no ?

Reply:
It is true that we do not allow `_shouldLockSetToken` to be `false` for the `OptimisticAuctionRebalanceExtension`, but this is not the case for the `AuctionRebalanceExtension`